### DISCUSSION TITLE
Run the super-linter container Locally.

### DISCUSSION THREAD

**Discussion by shivakumar-ralla** [super-linter/super-linter#2924] [Q&A]
Hi Team,

we try to run [Super-Linter Docker container](https://github.com/github/super-linter/blob/main/docs/run-linter-locally.md)  locally in our branch , super-linter linter applied entire repo level , we followed according to the steps in the above link , applied VALIDATE_ALL_CODEBASE to false to run linter on staged(changed) files , but after applying also showing the same result, linter applying on repo level instead of changed files.

Below are the docker steps followed to run locally , both resulting the same linters are applying complete repo level of each files.

    docker run -e RUN_LOCAL=true -v ${PWD}:/tmp/lint github/super-linter                    

    docker run  -e RUN_LOCAL=true VALIDATE_ALL_CODEBASE=false -v ${PWD}:/tmp/lint github/super-linter             
    
We want to validate linter only on changed files in local , can yo please provide any steps if I am missing anything.

**Comment by ferrarimarco** [comment]
Can you please file an issue about this so we can investigate?
